### DISCUSSION TITLE
How do I bump the version for a grammar?

### DISCUSSION THREAD

**Discussion by joelspadin** [tree-sitter/tree-sitter#3810] [Q&A] [answered]
https://github.com/tree-sitter/tree-sitter/issues/3320 indicates that the tree-sitter CLI will have a command to bump the version, but I cannot find it. Is this implemented yet, and if not, what is the best way to update the version number for a grammar?

If I manually update the version in either package.json or tree-sitter.json, then `tree-sitter init -u` seems like the most likely command to push that change to all the other bindings files (Cargo.toml, Makefile, etc.), but that only works if I manually delete all of the files it would generate first.

**Comment by ObserverOfTime** [comment, marked as answer]
#3786
